Sources/Amplitude/Configuration.swift

```diff
@@ -38,7 +38,7 @@ public class Configuration {
         apiKey: String,
         flushQueueSize: Int = Constants.Configuration.FLUSH_QUEUE_SIZE,
         flushIntervalMillis: Int = Constants.Configuration.FLUSH_INTERVAL_MILLIS,
-        instanceName: String = Constants.Configuration.DEFAULT_INSTANCE,
```

Comment:
Not sure I understand this change. Why do we need to handle "" vs not setting `instanceName`?

Reply:
Before:
`Configuration(apiKey)`  -> `instanceName == Constants.Configuration.DEFAULT_INSTANCE`
`Configuration(apiKey, instanceName = "")`  -> `instanceName == ""`
I think, it is an issue.

Now:
`Configuration(apiKey)`  -> `instanceName == Constants.Configuration.DEFAULT_INSTANCE`
`Configuration(apiKey, instanceName = "")`  -> `instanceName == Constants.Configuration.DEFAULT_INSTANCE`

Reply:
Ok got it. I thought it was needed for Ampli support.

Reply:
For posterity:
https://github.com/amplitude/Amplitude-JavaScript/blob/4cbe557a81ca981d03e140bebed6134c49595a5e/src/amplitude-client.js#L38